inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -366,11 +366,5 @@ auto_renew_certificates: false
 # First Monday of each month
 # auto_renew_certificates_systemd_calendar: "Mon *-*-1,2,3,4,5,6,7 03:{{ groups['kube_control_plane'].index(inventory_hostname) }}0:00"
 
-# kubeadm patches path
-kubeadm_patches:
-  enabled: false
-  source_dir: "{{ inventory_dir }}/patches"
-  dest_dir: "{{ kube_config_dir }}/patches"
-
```

Comment:
Should we consider replacing this with what `roles/kubernetes/kubeadm_common/defaults/main.yml` changed?

Reply:
I don't like the inventory/sample very much (see #10697 for some reasons) and I'd rather keep one source of truth and of documentation (= `defaults` folder).
What would I put there in place in your opinion ? Copy the default and comments of kubeadm_common/defaults/main.yml ?
(OTOH I missed the rest of the kubeadm_patches stuff in the inventory I'll remove that)

Reply:
I agree with you, but strangely enough, if I have a problem with Kubernetes, I intuitively think that I should go to https://kubernetes.io/docs/home/ first, but when I have a problem with Kubespray, the first thing I look for is the source code or an example, not https://kubespray.io.

I don't know if anyone else has similar problems, but when I implemented Ubuntu 24.04 support a few months ago, the documentation presentation wasn't the first thing I looked for.

Perhaps the presentation of the `kubespray.io` documentation needs to be changed and the outdated content removed. I'd like to know what you think about this.

Reply:
Back to the original question, for now, I think I can copy the default and comments of `kubeadm_common/defaults/main.yml`

In the future, after rewriting the documentation, I think there are some variables that can be removed (including kubeadm_patches, etc.).

Reply:
> Perhaps the presentation of the `kubespray.io` documentation needs to be changed and the outdated content removed. I'd like to know what you think about this.

I think our docs are in dire need of reorganization, yup :smile: . As I see it, the best way to find something is by grepping in `roles`, 




> Back to the original question, for now, I think we can copy the default and comments of `kubeadm_common/defaults/main.yml` 

Let's do that then. :+1: 

> In the future

:tm: